docs/amko_tenancy.md

```diff
@@ -0,0 +1,38 @@
+# Tenancy support in AMKO
+
+This feature allows AMKO to create GSLB object in a user-specific tenant in Avi.The expected isolation and administrative restrictions of a multi-tenant architecture in NSX Advanced Load Balancer GSLB extend to AMKO
+
+## Tenant Context
+
+AVI non admin tenants primarily operate in 2 modes, **provider context** and **tenant context**.
+
+### Provider Context
+
+Service Engine Groups are shared with `admin` tenant. All the other objects like Virtual Services and Pools are created within the tenant. Requires `config_settings.se_in_provider_context` flag to be set to `True` when creating tenant. 
+
+### Tenant Context
+
+Service Engines are isolated from `admin` tenant. A new `Default-Group` is created within the tenant. All the objects including Service Engines are created in tenant context. Requires `config_settings.se_in_provider_context` flag to be set to `False` when creating tenant. 
+
+## Steps to enable Tenancy in AMKO
+
+In this example we will run AMKO in `billing` tenant.
+
+### 1. Install AKO in required Tenant.
```

Comment:
Its not clear from teh documentation whether AKO and AMKO deployments need to be in the same tenant or can be in different ones

Reply:
mentioned explicityly AKO and AMKO needs to be in same tenant